wiki/Beatmap_Editor/fr.md

```diff
@@ -1,18 +1,32 @@
-# Éditeur de Beatmap
+# L'éditeur de Beatmap
 
-![Le mode Compose](Compose/Compose.png "Le mode Compose")
+**L'éditeur de beatmap** est une interface graphique intégrée au jeu pour la création de [beatmaps](/wiki/Beatmaps).
+Il est séparé en quatre parties:
 
- **L'éditeur de beatmap** est l'interface graphique d'osu! pour créer des [beatmaps](/wiki/Beatmaps). Il est divisé en 4 modes :
+- [Compose](Compose) : L'endroit où se déroule la majorité du processus de création de beatmap.
 
```

Comment:
there should be no empty lines between list items (this makes them render double-spaced)